markets/bfp-market/test/helpers.ts

```diff
@@ -23,7 +24,7 @@ declare module 'ethers' {
     [customInspectSymbol]?: () => string;
   }
 }
-ethers.BigNumber.prototype[customInspectSymbol] = function (this: ethers.BigNumber) {
+hre.ethers.BigNumber.prototype[customInspectSymbol] = function (this: ethers.BigNumber) {
```

Comment:
🤔 What's this for again and why the change?

Reply:
I think it's related to cannon using viem internally. I think that's when it stopped working. I guess now they grab ethers from hre instead?